packages/yew/Cargo.toml

```diff
@@ -89,16 +89,13 @@ trybuild = "1"
 
 [dev-dependencies.web-sys]
 version = "0.3"
-features = [
-  "ShadowRootInit",
-  "ShadowRootMode",
-  "HtmlButtonElement"
-]
+features = ["ShadowRootInit", "ShadowRootMode", "HtmlButtonElement"]
 
 [features]
 ssr = ["dep:html-escape", "dep:base64ct", "dep:bincode"]
 csr = []
 hydration = ["csr", "dep:bincode"]
+enable-interning = ["wasm-bindgen/enable-interning"]
```

Comment:
Can you please remove this feature? (see above discussion)
```suggestion
```

If you would like to also add documentation for interning, it should go in [optimizations docs](https://yew.rs/docs/next/advanced-topics/optimizations)